### DISCUSSION TITLE
Multihunter

### DISCUSSION THREAD

**Discussion by 0Fud** [adipiciu/Travian-scripts#77] [General]
Has anyone successfully used this Travian automation script without being penalized or banned by the multihunter? I've tried it on two different accounts and got punished both times with a 33% reduction in all building levels. Does anyone have tips or tricks to avoid detection? Would love to hear if someone managed to fly under the radar!

**Comment by 0Fud** [comment]
P.S. I've been using these scripts: Travian Task Queue for Travian 4, Travian Resource Bar Plus, and Travian Wave Builde

**Comment by Qussayyon** [comment]
Hi Multihunter,

Do you think someone would admit using these scripts here in public place?
Do you think it wise thing to admit?
You can always see users reporting bugs and so on, and no one report banning, not that they are getting banned but it seems that it's not so frequent that need to be reported.
How you are sure that Travian is implementing correct method to detect the 'illegal' use of userscripts?
How can any user of these script know if you are genuine user or you are from Travian team?

For me: without these scripts, especially TRB, I would not play Travian at all. For me it's not about automating building, But the fact that the script gives great and fast overview that is not possible even with Travian Plus.
For others they might think using automated building and queue is kinda void the point of playing in first place!

Kind regards
Genuine user of Travian and Travian scripts.
